app.rb

```diff
@@ -0,0 +1,164 @@
+# rubocop:disable Metrics/MethodLength, Lint/MissingCopEnableDirective
+# rubocop:disable Metrics/CyclomaticComplexity
+require_relative 'book'
+require_relative 'person'
+require_relative 'students'
+require_relative 'teacher'
+require_relative 'rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rental = []
+  end
+
+  # Create menu that allows to do his choice
+  def menu_showing
+    puts "\n"
+    puts 'Press 1 to show the list of books'
+    puts 'Press 2 to show the list of people'
+    puts 'Press 3 to add a person'
```

Comment:
When I select the option to add a person, I am prompted to `Press 1` to add a `teacher` and `press 2` to add a `student`.
Yet when I pick option 1 it's for creating a student, and option 2 is for creating a teacher.

- Kindly fix that to make sure that the option I pick is aligned with the person that is supposed to be created.